### ISSUE TITLE
Release version 0.6

### ISSUE THREAD

**Issue by naomijub** [kulkalkul/bevy_mod_billboard#25]
Main seems to be using version [0.6](https://github.com/kulkalkul/bevy_mod_billboard/blob/main/Cargo.toml#L3), but released versions go only up until [0.5.1](https://lib.rs/crates/bevy_mod_billboard/versions)

**Comment by kulkalkul** [comment]
#22
Unfortunately I couldn't find the time to investigate the issue. You can pin version to the main branch commit the time being.

**Comment by kulkalkul** [comment]
I should've released this way earlier, I got swamped with my own project, wasn't in a good mood most of the time and procrastinated it unnecessarily. Sorry about that!

Now that #26 is there, I give up on solving the performance regression anytime soon. By the time I'm available to look into it (3-6 months I hope) I think bevy might have its own billboarding solution implemented. If not, I'll look into it then (or make a PR for bevy's billboarding solution, which was something I wanted to do but couldn't).

In any case, I released the 0.6, will release the 0.7 for 0.14 after I review #26 when 0.14 is out 🥳
